_sass/_uswds-custom-styles.scss

```diff
@@ -0,0 +1,164 @@
+// ==========================================================================
+
+// ==========================================================================
+// Header
+// ==========================================================================
+.usa-header {
+
+  .usa-logo__text a {
+    @include add-background-svg("DHS_4R_Blue", "../../img");
+    background-size: cover;
+    display: inline-block;
+    // px-to-rem() is a built-in USWDS function.
+    height: px-to-rem(31px);
+    width: px-to-rem(106px);
+
+    @include at-media($theme-header-min-width) {
+      height: px-to-rem(72px);
+      width: px-to-rem(247px);
+    }
+  }
+}
+
+
+// ==========================================================================
+// Nav
+// ==========================================================================
+.usa-nav {
+  background-color: $c-dhs-blue;
+  color: color("white");
+
+  &__primary {
+
+    // The list containing menu items.
+    &-item {
+      border-top-color: $c-dhs-blue-40;
+    }
+
+    // Menu links on mobile and desktop
+    a,
+    button {
+      color: color("white");
+
+      &:hover {
+        color: color("white");
+        background-color: $c-dhs-blue-60;
+      }
+
+      @include at-media($theme-header-min-width) {
+        &:hover {
+          background-color: transparent;
+        }
+      }
+    }
+
+    // Need to override USWDS.
+    .usa-current {
+      color: color("white");
+
+      // Border on current link
+      &::after {
+        background-color: currentColor !important;
+        border-radius: radius(0);
+      }
+    }
+
+    // The border on the rest of the primary links on desktop hover.
+    .usa-nav__link:hover::after {
+      @include at-media($theme-header-min-width) {
+        background-color: currentColor !important;
+      }
+    }
+
+    // Main menu buttons
+    button[aria-expanded="false"] {
+      // This mixin allows us to define our own path for our custom images.
+      @include add-background-svg("plus-white", "../../img");
+
+      @include at-media($theme-header-min-width) {
+        &,
+        &:hover {
+          @include add-background-svg("angle-arrow-down-white");
+        }
+      }
+    }
+
+    button[aria-expanded="true"] {
+      @include add-background-svg("minus-white", "../../img");
+
+      @include at-media($theme-header-min-width) {
+        background-color: $c-dhs-light-blue-70;
+        @include add-background-svg("angle-arrow-up-white");
+      }
+    }
+  }
+
+  &__submenu-item {
+    border-top-color: $c-dhs-blue-40;
+  }
+}
+
+//
+// Secondary Nav (Search)
+// ==========================================================================
+
+// Remove these if you're adding secondary links.
+.usa-nav__secondary {
+  @include at-media($theme-header-min-width) {
+    bottom: px-to-rem(80);
+    margin-top: 0;
+  }
+
+  .usa-search {
+    @include at-media($theme-header-min-width) {
+      margin-top: 0;
+    }
+  }
+}
```

Comment:
7. Removed the spacing from the secondary nav (search) because it's just search for now.